### PR TITLE
Feat/jc/add signing in service

### PR DESCRIPTION
**Pull request by kochamdziekana**

### CODE REVIEW

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice.Data/Entities/Users/Administrator.cs [hunk 1]
```diff
@@ -1,5 +1,8 @@
-﻿namespace W4SRegistrationMicroservice.Data.Entities.Users
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace W4SRegistrationMicroservice.Data.Entities.Users
 {
+    [Table("Administrators")]
```
Comment (Crustacean1) [inline W4SRegistrationMicroservice/W4SRegistrationMicroservice.Data/Entities/Users/Administrator.cs:5]: Unnecessary, but ok

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice.Data/Entities/Users/User Settings/Roles.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace W4SRegistrationMicroservice.Data.Entities.Users.User_Settings
```
Comment (Crustacean1) [inline W4SRegistrationMicroservice/W4SRegistrationMicroservice.Data/Entities/Users/User Settings/Roles.cs:7]: Wierd namespace, mixing conventions, but leave it for now

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs [hunk 3]
```diff
@@ -19,11 +24,14 @@
 ConfigureLogger(builder.Host);
 
 ConfigureUserbaseDbContext(builder.Services, builder.Configuration.GetConnectionString("W4SRegistrationUserbase"));
+ConfigureValidators(builder.Services, builder.Configuration);
```
Comment (Crustacean1) [inline W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs:27]: This should be done in gateway only

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs [hunk 4]
```diff
@@ -19,11 +24,14 @@
...
 
 ConfigureServices(builder.Services);
 
 var app = builder.Build();
 
+app.UseAuthentication();
```
Comment (Crustacean1) [inline W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs:33]: As well as this

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs [hunk 5]
```diff
@@ -51,10 +59,42 @@ void ConfigureLogger(ConfigureHostBuilder host)
         .ReadFrom.Configuration(ctx.Configuration));
 }
 
+void ConfigureValidators(IServiceCollection services, IConfiguration configuration)
```
Comment (Crustacean1) [inline W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs:62]: What is purpose of this function???

**Review by Crustacean1 — APPROVED** [review]

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs (hunk 3 above)
Comment (kochamdziekana) [reply W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs:27]: Yes, it should be and will be

File: W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs (hunk 4 above)
Comment (kochamdziekana) [reply W4SRegistrationMicroservice/W4SRegistrationMicroservice/Program.cs:33]: Yes

**Pull request merged**